web/packages/plugins/shared/common/src/components/Pickers/BlueprintPicker.tsx

```diff
@@ -1,114 +1,43 @@
 // @ts-ignore
 import React, { useContext, useState } from 'react'
 import { BlueprintEnum } from '../../utils/variables'
-import { ApplicationContext, DmssAPI, Modal, useDataSources } from '../../index'
-import { IDataSources } from '../../hooks/useDataSources'
-import { AuthContext } from '@dmt/common'
+import { Modal, TreeNode, TreeView } from '../../index'
 import { Input } from '@equinor/eds-core-react'
-import { FaSpinner } from 'react-icons/fa'
-import styled from 'styled-components'
 
-type BlueprintPickerProps = {
-  onChange: Function
-  formData: any
-  uiSchema?: any
-  blueprintFilter?: BlueprintEnum
-}
-
-export const Spinner = styled(FaSpinner)`
-  animation: spin infinite 1s linear;
-
-  @keyframes spin {
-    from {
-      transform: rotate(0deg);
-    }
-    to {
-      transform: rotate(360deg);
-    }
-  }
-`
-
-export const Selector = ({ setShowModal, onChange, blueprintFilter }: any) => {
-  // @ts-ignore-line
-  const { token } = useContext(AuthContext)
-  const dmssApi = new DmssAPI(token)
-  const dataSources: IDataSources = useDataSources(dmssApi)
-  const application = useContext(ApplicationContext)
-
-  const onSelect = (value: string) => {
-    setShowModal(false)
-    onChange(value)
-  }
-
-  return (
-    <div />
-    // <Tree
-    //   state={index.models.tree.models.tree}
-    //   operations={index.models.tree.operations}
-    // >
-    //   {(renderProps: TreeNodeRenderProps) => {
-    //     const { actions, nodeData } = renderProps
-    //
-    //     if (nodeData.meta.type === blueprintFilter) {
-    //       const onClick = () => {
-    //         onSelect(`${renderProps.path}/${nodeData.title}`)
-    //       }
-    //       return (
-    //         <div
-    //           style={{ display: 'flex', flexDirection: 'row' }}
-    //           onClick={onClick}
-    //         >
-    //           {renderProps.iconGroup(() => onClick())}
-    //           {nodeData.title}
-    //         </div>
-    //       )
-    //     } else {
-    //       return (
-    //         <div
-    //           onClick={() => handleOpenOrExpand(renderProps)}
-    //           style={{ display: 'flex', flexDirection: 'row' }}
-    //         >
-    //           {renderProps.iconGroup(() => handleOpenOrExpand(renderProps))}
-    //           {nodeData.title}
-    //           {nodeData.isLoading && (
-    //             <small style={{ paddingLeft: '15px' }}>Loading...</small>
-    //           )}
-    //         </div>
-    //       )
-    //     }
-    //   }}
-    // </Tree>
-  )
-}
-
-export const BlueprintPicker = (props: BlueprintPickerProps) => {
+export const BlueprintPicker = (props: {
+  onChange: (type: string) => void
+  formData: string
+  blueprintFilter?: string
```

Comment:
i don't think we need blueprintFilter as a prop, since we always want to have blueprintFilter=BlueprintEnum.BLUEPRINT in the blueprintpicker.

Reply:
done